docs/guides/cron.rst

```diff
@@ -0,0 +1,30 @@
+Cron
+====
+
+If you setup cron correctly as described in :doc:`/admin/cron` 
+special hooks will be triggered so you can register for these hooks from your own code.
+
+The example below registers a function for the daily cron.
+
+.. code:: php
+	
+	function my_plugin_init() {
+	    elgg_register_plugin_hook_handler('cron', 'daily', 'my_plugin_cron');
+	}
+	
+
+If timing is important in your cron hook be advised that the functions
+are executed in order of registration. This could mean that your function may
```

Comment:
that is bad practice for cron... because if all hooks did that, they would end up with a problem. Using the time param is safer...